cms/djangoapps/models/settings/course_metadata.py

```diff
@@ -135,11 +135,6 @@ def get_exclude_list_of_fields(cls, course_key):
         if not COURSE_ENABLE_UNENROLLED_ACCESS_FLAG.is_enabled(course_key=course_key):
             exclude_list.append('course_visibility')
 
-        # Do not show "Create Zendesk Tickets For Suspicious Proctored Exam Attempts" in
-        # Studio Advanced Settings if the user is not edX staff.
-        if not GlobalStaff().has_user(get_current_user()):
-            exclude_list.append('create_zendesk_tickets')
```

Comment:
Removing this because it was preventing non edX staff from making permanent changes to this field. By removing this, course staff will also be able to view the field as a deprecated setting.